esmvaltool/recipes/recipe_autoassess_landsurface_soilmoisture.yml

```
@@ -4,17 +4,14 @@
 documentation:
   description: |
     Recipe that runs the Autoassess Land-surface assessment area diagnostic.
-    Climatological files are stored externally to avoid overloading the
-    ESMValTool source. See /gws/nopw/j04/esmeval/autoassess_specific_files
-    (on JASMIN).
 
   authors:
     - predoi_valeriu
     - sellar_alistair
 
   title: Land-surface diagnostic that computes soilmoisture indices (from Autoassess).
 
-  references:
+  observation_references:
```

```suggestion
  references:
```
`references` is the key that is supported by the mechanism for reporting provenance. See also https://docs.esmvaltool.org/projects/ESMValCore/en/latest/recipe/overview.html#recipe-section-documentation and https://docs.esmvaltool.org/en/latest/community/diagnostic.html#recording-provenance